modules/querying/pages/func/query-user-defined-functions.adoc

```diff
@@ -14,6 +14,11 @@ There are two ways to modify these files to add user-defined functions to GSQL:
 
 * Store the files in a GitHub repository, and configure GSQL to read from the repository.
 * Use `GET` and `PUT` commands to download, modify, and store the files locally.
+** The `GET` command requires the xref:tigergraph-server:user-access:access-control-model.adoc#_privileges[`WRITE_FILE` privilege].
+** The `PUT` command requires the `READ_FILE` privilege.
```

Comment:
PUT means write. So, should it be "WRITE_FILE" privilege for PUT and "READ_FILE" privilege for GET command?